ToolBox/GoogleDownloader/GoogleDataDownloader.cs

```diff
@@ -122,7 +122,7 @@ private IEnumerable<BaseData> GetDaily(Symbol symbol, Resolution resolution, Dat
         // p = period in days
         // ts = start time
         // Strangely Google forces CHLO format instead of normal OHLC.
-        private const string UrlPrototypeMinuteOrHour = @"http://www.google.com/finance/getprices?q={0}&i={1}&p={2}d&f=d,c,h,l,o,v&ts={3}";
+        private const string UrlPrototypeMinuteOrHour = @"https://finance.google.com/finance/getprices?q={0}&i={1}&p={2}d&f=d,c,h,l,o,v&ts={3}";
```

Comment:
Interestingly, this change appears to work and the old one doesn't work. C'mon google!